### DISCUSSION TITLE
Should focus_on_show be the default?

### DISCUSSION THREAD

**Discussion by kbarros** [JuliaPlots/Makie.jl#2806] [General]
By default, the GLMakie plot window appears in the background. This setting can be changed with `GLMakie.activate!(focus_on_show=true)`. Should this be the default? For non-experts, it's easy to miss that an interactive plotting window has been opened in background. This is especially true in the context of a VSCode editor, when another inline plot has already appeared.

**Comment by Sagnac** [comment]
in scripts and packages i do tend to set it to `true`, especially if the figure is expected to be interacted with, but when working from the REPL prototyping and testing stuff then having it steal focus from the command line can get kind of annoying especially if you're adding stuff to a plot and building it step-by-step; i think `focus_on_show = false` is currently a good default considering this type of workflow in the initial stages
